packages/core/src/codewhisperer/util/editorContext.ts

```diff
@@ -202,18 +203,26 @@ function logSupplementalContext(supplementalContext: CodeWhispererSupplementalCo
         return
     }
 
-    let logString = `CodeWhispererSupplementalContext:
-    isUtg: ${supplementalContext.isUtg},
-    isProcessTimeout: ${supplementalContext.isProcessTimeout},
-    contentsLength: ${supplementalContext.contentsLength},
-    latency: ${supplementalContext.latency},
-`
+    let logString = indent(
+        `CodeWhispererSupplementalContext:
+        isUtg: ${supplementalContext.isUtg},
+        isProcessTimeout: ${supplementalContext.isProcessTimeout},
+        contentsLength: ${supplementalContext.contentsLength},
+        latency: ${supplementalContext.latency}
+        strategy: ${supplementalContext.strategy}`,
+        4,
+        true
+    ).trimStart()
```

Comment:
Nice! Thanks for thinking about the log format, that helps with troubleshooting for both users and devs.